### PR TITLE
Blog post for April 23

### PR DESCRIPTION
**Pull request by valgrimm**
Authored last week, forgot to PR . . .

### CODE REVIEW

File: posts/inside-rust/2020-04-23-Governance-wg.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Governance Working Group Update: Meeting 23 April 2020"
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
```
Comment (XAMPPRocky) [inline posts/inside-rust/2020-04-23-Governance-wg.md:4]: It's missing the author field. 😄

Comment (valgrimm) [reply posts/inside-rust/2020-04-23-Governance-wg.md:4]: Is that really needed?

Comment (XAMPPRocky) [reply posts/inside-rust/2020-04-23-Governance-wg.md:4]: Currently yes it's required in the software. If you don't want to sign your name the team field is actually optional so you could just change that to `author`.
```suggestion
author: The Governance WG <https://github.com/rust-lang/wg-governance>
```

**Comment by XAMPPRocky** [comment]
Thank you for your PR!

**Pull request merged**